Animated.js

```diff
@@ -682,11 +682,19 @@ class AnimatedVec2 extends AnimatedWithChildren {
 class AnimatedInterpolation extends AnimatedWithChildren {
   _parent: Animated;
   _interpolation: (input: number) => number | string;
+  _listeners: {[key: number]: ValueListenerCallback};
+  _parentListener: number;
 
   constructor(parent: Animated, interpolation: (input: number) => number | string) {
     super();
     this._parent = parent;
     this._interpolation = interpolation;
+    this._listeners = {};
+    this._parentListener = parent.addListener(() => {
```

Comment:
most of the time, there's --not-- going to be a listener here. So this is going to be very expensive to setup all those listeners. I would do it on the fly when the first addListener is being called